sdk/deepbook-v3/src/transactions/deepbook.ts

```diff
@@ -35,7 +36,6 @@ export class DeepBookContract {
 			payWithDeep = true,
 		} = params;
 
-		tx.setGasBudget(GAS_BUDGET);
```

Comment:
We need to set a gas budget due to discrepancy between dry run estimation and the actual execution for some transactions, should we make it clear in SDK docs instead of the contracts?

Reply:
Why does this happen, and how big is the discrepancy?  The SDK adds some overhead to the estimate, but if it's not big enough this might be something we should fix in the ts sdk